lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
+                                       default_value: 'bootstrap',
+                                       verify_block: proc do |value|
+                                         raise "Please pass a valid command. Use one of the following: build, bootstrap, update" unless %w(build bootstrap update).include? value
+                                       end),
           FastlaneCore::ConfigItem.new(key: :update,
```

Comment:
ah! my bad, i didn't have that context that it was just added last week, yeah. probably better to get rid of it now than have it hanging around! apologies for the run-around @uny, you can go ahead and remove it, squash it and then we will merge it! 

we can point people here if they are running into issues.